src/nwb_benchmarks/globals.py

```diff
@@ -1,6 +1,6 @@
 import pathlib
 
-MACHINE_FILE_VERSION = "1.3.0"
+MACHINE_FILE_VERSION = "1.4.1"
 DATABASE_VERSION = "2.0.0"
 
 CACHE_DIR = pathlib.Path.home() / ".cache" / "nwb_benchmarks"
```

Comment:
Can we rename this "APP_CACHE_DIR" or something? Just because there are two different meanings of "cache" settings now.

Reply:
A lot of the current usages of the `CACHE_DIR` are small files (results, environments, machines, logs)

Could we consolidate those to the 'home' directory or would you like two separate 'cache' directories, one for the 'app internals' and one for 'benchmarks'

Reply:
Either way is OK with me, but I would like to not auto-delete the logs after every run since that is useful for debugging. The uploaded results might be nice to keep around too, but I could go either way now that we have the logs. I don't know that we need to cache the environments and machines.

Reply:
Nothing from 'that cache' is intended to be deleted, only the 'benchmarks' ones